OrbitCore/CaptureData.h

```diff
@@ -125,11 +125,19 @@ class CaptureData {
     return tracepoint_info_manager_->Get(key);
   }
 
+  [[nodiscard]] const TracepointInfoManager* GetTracepointInfoManager() const {
+    return tracepoint_info_manager_.get();
```

Comment:
Just out of curiosity, what is the reason, why the `tracepoint_info_manager-` and the `tracepoint_event_buffer_` are explicitly allocated on the heap using a unique pointer. Can't they just be *value* member variable of the `CaptureData`?

Reply:
Thank you for asking. A similar decision was made for the callstacks. The motivation was that  std::unique_ptr<> allows to move and copy data easier (especially relevant if there is stored an absl::Mutex inside).